CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v5.11.0(unreleased)
```

Comment:
i think your release version should be 5.10.1 as its just a patch

Reply:
ignore this as another PR is extending the support of Hypervsior Manager for API 1000/1200

Reply:
I believe, I should not do any change based on above comments. Please correct me if I am wrong.

Reply:
thats correct